OCA-UFCG/oca-frontend
# Fix/fix first faulty post req

This PR fixed the bug of the wrong POST request, more specifically the first one that is made when the home page is refreshed. That was due to a wrong use of the "default" value of the request.

## how to test it
1. go to '/'
2. watch the application console (server side) and verify that the first POST request is made for "cisternas" visualization.

## Screenshots

### wrong post request being made
![image](https://github.com/user-attachments/assets/034252f7-7713-4ef8-83ee-78b0f40c5945)

### fixed post requests log
![image](https://github.com/user-attachments/assets/1c6142ce-174c-4ad9-b473-5c34fd126687)
